app/main.py

```diff
@@ -1 +1,55 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self._health = health
+        self.hidden = hidden
+
+        Animal.alive.append(self)
+
+    def __str__(self) -> str:
+        return (
+            "{"
+            f"Name: {self.name}, "
+            f"Health: {self.health}, "
```

Comment:
The `__str__` method is trying to access `self.health`, but `health` is a property. This is correct usage, but ensure that the property is functioning as expected.